### PR TITLE
fix(ingest/protobuf): Improve String Handling for Multilingual Support in Descriptions

### PR DESCRIPTION
**Pull request by eunseokyang**
## Checklist

- [ ] The PR conforms to DataHub's [Contributing Guideline](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [ ] Links to related issues (if applicable)
- [ ] Tests for the changes have been added/updated (if applicable)
- [ ] Docs related to the changes have been added/updated (if applicable). If a new feature has been added a Usage Guide has been added for the same.
- [ ] For any breaking change/potential downtime/deprecation/big changes an entry has been made in [Updating DataHub](https://github.com/datahub-project/datahub/blob/master/docs/how/updating-datahub.md)

## Description
Support for Multilingual Characters: By returning the original string directly, we ensure that all characters, including those outside the ISO-8859-1 range (e.g., Korean, Chinese, Japanese), are preserved and correctly displayed.

Upon further investigation on [this issue](https://github.com/protocolbuffers/protobuf/issues/4691), it has been identified that the root cause of the broken UTF-8 characters is user error, specifically related to incorrect handling of character encodings in user input or data processing.

AS-IS
image

TO-BE
image

## Summary by CodeRabbit

- **New Features**
	- Enhanced comment handling to support UTF-8 characters, improving compatibility with various text formats.
  
- **Bug Fixes**
	- Removed unnecessary intermediate processing, streamlining comment concatenation.

- **Refactor**
	- Simplified the implementation of the comment processing method for better performance and readability.

### CODE REVIEW

**Comment by david-leifker** [comment]
Thanks for this! It has been awhile since I wrote that code, but it was definitely not working with UTF8 characters at one time. I'd be happy to merge this PR if a unit test using a UTF8 character could be added.

**Comment by eunseokyang** [comment]
@david-leifker I have added a unit test for verifying UTF-8 characters in protobuf comments. PTAL! https://github.com/datahub-project/datahub/pull/10975/commits/1f5fd5b0ab02a9bcaf1e22b4b7fd8944b8664faf

File: metadata-integration/java/datahub-protobuf/src/test/java/datahub/protobuf/ProtobufUtilsTest.java [hunk 1]
```diff
@@ -44,4 +45,18 @@ public void registryTest() throws IOException, IllegalArgumentException {
...
+    DescriptorProtos.SourceCodeInfo.Location location = DescriptorProtos.SourceCodeInfo.Location.newBuilder()
+        .addAllLeadingDetachedComments(Arrays.asList("/* Emoji 😊 */", "/* Accented é */"))
+        .setLeadingComments("/* Chinese 你好 */\n// Russian Привет")
+        .setTrailingComments("// Korean 안녕")
+        .build();
+
+    String result = ProtobufUtils.collapseLocationComments(location);
+    String expected = "Emoji 😊 */\nAccented é */\nChinese 你好 */\nRussian Привет\nKorean 안녕";
+
+    assertEquals(expected, result);
```
Comment (happyprg) [inline metadata-integration/java/datahub-protobuf/src/test/java/datahub/protobuf/ProtobufUtilsTest.java:60]:
```suggestion
    String actual = ProtobufUtils.collapseLocationComments(location);
    String expected = "Emoji 😊 */\nAccented é */\nChinese 你好 */\nRussian Привет\nKorean 안녕";

    assertEquals(expected, actual);
```

**Comment by david-leifker** [comment]
Thanks!

**Review by david-leifker — APPROVED** [review]

**Pull request merged**